README.md

```diff
@@ -1 +1,128 @@
-# grid3_client_go
\ No newline at end of file
+# **grid3_go:**
+
+grid3_go is a go client created to interact with threefold grid. It should manage CRUD operations for deployments on the grid.
+
+- ## **Grid3_go flow:**
+
+  1. the deployment manager will be initialized with identity/network information.
+  2. deployment manager will expose the `deploy` which the user will call it and give it old/new deployments
+  3. `deploy` method will do the following
+     - internally call `calculateChanges` method which
+       1. loads old deployments using their ids from teh grid
+       2. determine which things needs to be created, updated and deleted
+     - will take the sutable action for each operation to create, update and delete
+     - waits on them and report the state
+  4. For applying the changes we have subi and node pkgs which creates contracts/deployments on the grid
```

Comment:
please use backticks around subi and node and use the full word `package` instead of `pkg`, I also got very confused thinking you mean the node runtime :D